src/herder/test/TransactionQueueTests.cpp

```diff
@@ -1783,11 +1946,13 @@ TEST_CASE("transaction queue starting sequence boundary",
     }
 }
 
-TEST_CASE("transaction queue with fee-bump", "[herder][transactionqueue]")
+void
+testTxQueueFeeBump(bool limit)
```

Comment:
nit: `limitSourceAccounts`? It's unclear what exactly is limited here.

Reply:
agreed, updated.

Reply:
Please also update `testReplaceByFee` as well